tests/page-objects/formElements2.ts

```diff
@@ -0,0 +1,199 @@
+import { Page,Locator,expect} from '@playwright/test'
```

Comment:
change the file name to something more unique. or change the name of the other one.